HISTORY.md

```diff
@@ -1,5 +1,13 @@
 # Version History
 
+## 1.1.13 / 2023-03-07
```

Comment:
any reason to have a double version jump?

Reply:
No, I think I was just tired that day 😅